### PR TITLE
Docker: add darwin/arm and linux/arm for target platforms

### LINKED ISSUE #557: Build Mac ARM and Linux ARM Docker Images

### PR DESCRIPTION
**Pull request by saraswatpuneet**
## Details
- [x] Add support for darwin/arm
- [x] Add support for linux/arm

Closes #557

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -113,7 +113,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64, darwin/arm64
```
Comment (wilwade) [inline .github/workflows/release.yml:116]: Is this really all we need? Is that because the `node:20` images support those platforms?

**Review by mattheworris — APPROVED** [review]

File: .github/workflows/release.yml (hunk 1 above)
Comment (saraswatpuneet) [reply .github/workflows/release.yml:116]: And node:20 supports these 

https://github.com/docker-library/official-images?tab=readme-ov-file#architectures-other-than-amd64


Not sure about Darwin arm but checking

Comment (saraswatpuneet) [reply .github/workflows/release.yml:116]: yep node supports darwin/arm https://hub.docker.com/_/node

Comment (saraswatpuneet) [reply .github/workflows/release.yml:116]: @wilwade  i do not see an arm for 20 

https://hub.docker.com/r/arm64v8/node/tags

Comment (saraswatpuneet) [reply .github/workflows/release.yml:116]: though i do not see darwin anywhere

**Comment by saraswatpuneet** [comment]
@wilwade  confirmed no Darwin arm images for node20 yet , should we release arm Linux for now ? Or come back at it in few months

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -113,7 +114,7 @@ jobs:
         uses: docker/build-push-action@v5
         with:
           context: .
-          platforms: linux/amd64
+          platforms: linux/amd64, linux/arm64/v8
```
Comment (saraswatpuneet) [inline .github/workflows/release.yml:117]: `linux/arm64/v8` works on darwin as well

**Review by wilwade — APPROVED** [review]
🎉

**Comment by saraswatpuneet** [comment]
https://hub.docker.com/r/projectlibertylabs/content-watcher-service/tags

**Pull request merged**